packages/fxa-settings/src/pages/Signin/SigninPushCodeConfirm/index.tsx

```diff
@@ -0,0 +1,168 @@
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Link, RouteComponentProps } from '@reach/router';
+import { RemoteMetadata } from '../../../lib/types';
+
+import AppLayout from '../../../components/AppLayout';
+import DeviceInfoBlock from '../../../components/DeviceInfoBlock';
+import { FtlMsg } from '../../../../../fxa-react/lib/utils';
+import Banner, { BannerType } from '../../../components/Banner';
+
+// Reuse these images temporarily
+import monitorIcon from '../../../components/Settings/BentoMenu/monitor.svg';
+import relayIcon from '../../../components/Settings/BentoMenu/relay.svg';
+import vpnIcon from '../../../components/Settings/BentoMenu/vpn-logo.svg';
+import checkmarkIcon from './greencheck.svg';
+import { LinkExternal } from '@fxa/shared/react';
+
+export type SigninPushCodeConfirmProps = {
+  authDeviceInfo: RemoteMetadata;
+  handleSubmit: () => void;
+  sessionVerified: boolean;
+  isLoading: boolean;
+  errorMessage?: string;
+};
+
+const ProductPromotion = () => {
+  const products = [
+    {
+      icon: monitorIcon,
+      title: 'Mozilla Monitor',
+      description:
+        'Get notified if your information is involved in a data breach.',
+      link: 'https://monitor.firefox.com/',
+    },
+    {
+      icon: relayIcon,
+      title: 'Firefox Relay',
+      description:
+        'Keep your email address safe from spam and unwanted emails.',
+      link: 'https://relay.firefox.com/',
+    },
+    {
+      icon: vpnIcon,
+      title: 'Mozilla VPN',
+      description:
+        'Protect your internet connection and browse privately with Mozilla VPN.',
+      link: 'https://vpn.mozilla.org/',
+    },
+  ];
+
+  // Note this isn't localized yet since the UX will most likely change
+  return (
+    <div className="mt-8">
+      <h2 className="text-lg font-bold mb-6">
+        Explore products from Mozilla that protect your privacy
+      </h2>
+      {products.map((product, index) => (
+        <div
+          key={index}
+          className="flex items-center justify-center my-6 space-x-4"
+        >
+          <img
+            src={product.icon}
+            alt={`${product.title} Logo`}
+            className="w-12 h-12"
+          />
+          <div className="text-left max-w-xs">
+            <h3 className="text-md font-semibold mb-1">{product.title}</h3>
+            <p className="text-sm leading-relaxed">
+              {product.description}
+              <LinkExternal
+                href={product.link}
+                className="text-blue-500 underline"
+              >
+                {' '}
+                Learn more
+              </LinkExternal>
+            </p>
+          </div>
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const LoginApprovedMessage = () => {
+  return (
+    <div className="text-center mt-4">
+      <img
+        src={checkmarkIcon}
+        className="w-12 h-12 mx-auto mb-4"
+        alt="Checkmark Icon"
```

Comment:
Not for here but I think we don't need alt text for this since it'd be considered decorative, since the next line has "approved". On Connect Another Device we've got the TW class `before:content-circle-check` to just have it as a pseudo element.